dotCMS/src/integration-test/java/com/dotcms/scheduler/TestSyncronizedTask.java

```diff
@@ -0,0 +1,55 @@
+package com.dotcms.scheduler;
+
+import com.dotmarketing.exception.DotRuntimeException;
+import io.vavr.control.Try;
+
+public class TestSyncronizedTask extends DotSyncronizedTask{
+
+    private final static String SYSTEM_INCREMENT_KEY = "TestSyncronizedTaskKey";
+    private final static String TASK_RUNNING_KEY = "TestSyncronizedTaskRunning";
+    public static int getTotalIncrement() {
+        return Try.of(() -> Integer.parseInt((String) System.getProperty(SYSTEM_INCREMENT_KEY))).getOrElse(0);
+
+    }
+    
+    
+    
+    
+    final int myIncrement;
+    
+    public TestSyncronizedTask(int inc) {
```

Comment:
Codacy found an issue: [It is a good practice to call super() in a constructor](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6006494)